admin_guide/diagnostics_tool.adoc

```diff
@@ -0,0 +1,51 @@
+[[admin-guide-diagnostics-tool]]
+= Diagnostics Tool
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The `oc adm diagnostics` command launches a diagnostic pod that reports on
```

Comment:
launching a diagnostic pod is one thing it does but I wouldn't say that's the overview...

More like:
"... runs a series of checks for error conditions in the host or cluster. Specifically, it:"